UPGRADE-5.1.md

```diff
@@ -19,6 +19,13 @@ HttpFoundation
    `RedirectResponse::create()`, and `StreamedResponse::create()` methods (use
    `__construct()` instead)
 
+Messenger
+---------
+
+ * Deprecated AmqpExt transport. It has moved to a separate package. Run `composer require symfony/amqp-messenger` to use the new classes.
+ * Deprecated Doctrine transport. It has moved to a separate package. Run `composer require symfony/doctrine-messenger` to use the new classes.
```

Comment:
some nitpick. I would rename this one to `doctrine-dbal-messenger`. The Doctrine organization has many separate projects related to persistence. This bridge integrates with Doctrine DBAL, not with the whole Doctrine projects (having a single bridge for the whole Doctrine makes the dependency management a pain for `symfony/doctrine-bridge` btw, so if we ever have another transport using a different Doctrine project, I'd rather have a separate bridge for it)